assets/dev/js/frontend/handlers/section.js

```diff
@@ -1,3 +1,155 @@
+class BackgroundSlideshow extends elementorModules.frontend.handlers.Base {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				swiperContainer: '.elementor-section-background-slideshow',
+				swiperSlides: '.swiper-slide',
+			},
+			classes: {
+				swiperContainer: 'elementor-section-background-slideshow swiper-container',
+				swiperWrapper: 'swiper-wrapper',
+				swiperSlide: 'elementor-section-background-slideshow__slide swiper-slide',
+				swiperSlideInner: 'elementor-section-background-slideshow__slide__image',
+			},
+		};
+	}
+
+	getSwiperOptions() {
+		const elementSettings = this.getElementSettings();
+
+		const swiperOptions = {
+			grabCursor: false,
+			slidesPerView: 1,
+			slidesPerGroup: 1,
+			loop: 'yes' === elementSettings.background_slideshow_loop,
+			speed: elementSettings.background_slideshow_animation_speed,
+			autoplay: {
+				delay: elementSettings.background_slideshow_autoplay_speed,
+				disableOnInteraction: false,
```

Comment:
Is it really required?

Reply:
it is true by default, it will stop the autoplay when a user has their mouse on the section.
It is not necessary, it is a matter of specs (אפיון). Should be decided...